lib/react_on_rails/helper.rb

```diff
@@ -398,26 +398,25 @@ def build_react_component_result_for_server_streamed_content(
       component_specification_tag: required("component_specification_tag"),
```

Comment:
Can you check if all of these "required" calls can go away given that we use a more recent version of Ruby?

Reply:
Yes, it's not needed. Removed